torch/_inductor/index_propagation.py

```diff
@@ -173,7 +173,7 @@ def inner(*args, **kwargs):
                 for a in itertools.chain(args, kwargs.values())
                 if isinstance(a, IndexPropVar)
             ]
-            if not all(v.is_symbolic for v in var_arguments):
+            if len(var_arguments) == 0 or not all(v.is_symbolic for v in var_arguments):
```

Comment:
What this PR does is completely disable index propagation... `ops.constant` and `ops.index_expr` have no variable arguments, and if they fallback we will never construct a symbolic `IndexPropVar`.